scripts/requirements-extras.txt

```diff
@@ -16,4 +16,4 @@ lpc_checksum
 Pillow
 
 # can be used to sign a Zephyr application binary for consumption by a bootloader
-imgtool
+imgtool>=1.7.0.1
```

Comment:
This release won't exist, please use `1.7.1` we will hopefully release it today.